### DISCUSSION TITLE
App Hub Submission Guidelines - Proposal

### DISCUSSION THREAD

**Discussion by cooper-joe** [dhis2/notes#198] [Ideas] [answered]
There is an ongoing discussion about the need for clear guidelines for App Hub submissions. I have written a draft proposal of these guidelines: [App Hub Sumission Guidelines - Proposal](https://docs.google.com/document/d/1R3JTjdA7-jh3ZnMNhy695H2_TUDkvgXe1wfoSXA4EmE/edit?usp=sharing). Feedback is very welcome.

**Comment by varl** [comment, marked as answer]
These guidelines have been published: https://developers.dhis2.org/guides/app-hub-submission-guidelines
